modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -3089,26 +3089,31 @@
             "required": true,
             "description": "Scoping of appeal type for associated issues",
             "schema": {
-              "type": "string"
+              "type": "string",
+              "enum": [
+                "higher_level_reviews",
+                "notice_of_disagreements",
+                "supplemental_claims"
```

Comment:
We shouldn't add SC's to prod documentation just yet.

Reply:
RIGHT!